compiler/rustc_target/src/spec/mod.rs

```diff
@@ -2587,12 +2582,21 @@ impl Target {
         key!(supported_split_debuginfo, falliable_list)?;
         key!(supported_sanitizers, SanitizerSet)?;
         key!(default_adjusted_cabi, Option<Abi>)?;
-        key!(c_enum_min_bits, u64);
         key!(generate_arange_section, bool);
         key!(supports_stack_protector, bool);
         key!(entry_name);
         key!(entry_abi, Conv)?;
 
+        if let Some(s) = obj
+            .remove(&(stringify!(c_enum_min_bits)).replace("_", "-"))
```

Comment:
`stringify` and `replace` are probably unnecessary, wouldn't it be simpler to write `"c-enum-min-bits"` here directly?

Reply:
Also, pre-existing so not for this PR, but I'm surprised this just silently ignores deserialization failures...